### PR TITLE
STONEBLD-789: Disable .tekton gitops dir creation

### PR DESCRIPTION
**Pull request by Michkov**
Avoid deployment of Repository CR on external cluster by disabling the .tekton folder creation.

### PR acceptance criteria:

- [ ] Unit/Functional tests

  

- [ ] Documentation 

   

- [ ] Client Impact

### CODE REVIEW

**Review by johnmcollier — APPROVED** [review]
LGTM.

File: controllers/component_controller.go [hunk 1]
```diff
@@ -548,12 +545,6 @@ func (r *ComponentReconciler) generateGitops(ctx context.Context, req ctrl.Reque
...
 	}
 
-	// GenerateTektonBuild returns a sanitized error message
-	err = appservicegitops.GenerateTektonBuild(tempDir, *component, r.AppFS, gitOpsContext, gitopsConfig)
-	if err != nil {
-		log.Error(err, "unable to generate gitops build resources due to error")
-		return err
-	}
 	//Gitops functions return sanitized error messages
 	err = r.Generator.CommitAndPush(tempDir, "", gitOpsURL, mappedGitOpsComponent.Name, gitOpsBranch, "Generating Tekton resources")
```
Comment (johnmcollier) [inline controllers/component_controller.go:549]: Ah, missed this. Can you actually change this line here to say "Generated GitOps Resources"?

Comment (Michkov) [reply controllers/component_controller.go]: updated

**Review by sbose78 — APPROVED** [review]

**Pull request merged**